addons/mail/static/src/components/message/message.scss

```diff
@@ -45,9 +45,13 @@
     word-wrap: break-word;
     word-break: break-word;
 
-    * {
+    *:not(li):not(li div) {
```

Comment:
The `:not(li div)` is not working for me, but it is not needed either.

Reply:
It's not needed in Chrome and Firefox, but Safari needs it so that 1st leaf div in a list overflow rule doesn't impact the bullet point of list item.